quint/src/cliCommands.ts

```diff
@@ -515,8 +515,8 @@ function maybePrintWitnesses(verbosityLevel: number, outcome: Outcome, witnesses
 export async function runSimulator(prev: TypecheckedStage): Promise<CLIProcedure<TracingStage>> {
   const simulator = { ...prev, stage: 'running' as stage }
   const startMs = Date.now()
-  // Force disable output if `--out-itf` is set
-  const verbosityLevel = prev.args.outItf ? 0 : deriveVerbosity(prev.args)
+  // Set verbosity to 0 if --out-itf is set and no explicit --verbosity is provided
+  const verbosityLevel = prev.args.outItf && prev.args.verbosity === verbosity.defaultLevel ? 0 : deriveVerbosity(prev.args)
```

Comment:
Hmm cool idea, but I think this is not ideal as it is impossible to pass `--verbosity=2`, as 2 is the default level.

I think it is fine to break backwards compatibility and just get rid of this suppressing output feature completely. This was introduced before we had `--verbosity` (iirc) and `--verbosity` should now be the only way to control how much output we see.